### DISCUSSION TITLE
Canal del Sol [DO]

### DISCUSSION THREAD

**Discussion by Carlinhos027** [iptv-org/community#388] [Show and tell]
### Channel ID

CanaldelSol.do

### Stream URL

https://canaldelsoltv01.streamprolive.com/hls/live.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
Missing referer or something, link alone redirects to Google.

**Comment by freearhey** [comment]
https://canaldelsoltv01.streamprolive.com/hls/live.m3u8 (offline)
<img width="399" alt="image" src="https://user-images.githubusercontent.com/7253922/232256921-a689c259-e0dc-41fd-b0c5-fcef1a79b1d3.png">
